brave/brave-core
# (Wallet) Load deposit Web UI

## Resolves https://github.com/brave/brave-browser/issues/31495

Expose deposit Web UI on Android.

The new deposit page can be accessed by navigating to `brave://wallet/deposit-funds` or tapping on the FAB in the Wallet section and selecting `Deposit`.

<img width="698" alt="Screenshot 2023-07-14 at 12 01 24 PM" src="https://github.com/brave/brave-core/assets/3308503/cadff118-d4c7-450a-b99c-7a9ae3d0e81c">

<img width="698" alt="Screenshot 2023-07-14 at 12 02 09 PM" src="https://github.com/brave/brave-core/assets/3308503/ecf933cc-7d86-4f5c-a0b4-7d0bb143c52d">




## Known Issues

### Resources not found

There's a console error related to some missing SVG files, `checkbox-checked.svg` and `checkbox-unchecked.svg`. The error logged in the console is probably related to a caching issue and does not affect the look and feel as the icons are loaded correctly. More info in the Slack discussion: https://bravesoftware.slack.com/archives/C03A0MWJXP0/p1689269592129619


![DevTools_-_chrome___wallet_crypto_deposit-funds (1)](https://github.com/brave/brave-core/assets/3308503/f0e82a05-5a90-45b9-a9a5-450e9290cf7c)

### Scrolling stuttering

The performance is a bit choked when the asset list is loading. This issue will be fixed by https://github.com/brave/brave-core/pull/19306


## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Open the Wallet app
* Tap on the FAB
* Tap on Deposit
* Observe the deposit screen loading
* Select Solana from the list and tap continue
* Observe that the address is shown correctly
